tbx/project_styleguide/templates/patterns/molecules/footer-cta/footer-cta.html

```diff
@@ -0,0 +1,18 @@
+<div class="grid__footer-contact footer-cta">
+    {% include "patterns/atoms/motif-heading/motif-heading.html" with heading=contact_heading heading_level=2 classes="motif-heading--two motif-heading--static footer-cta__heading" %}
+
+    <p class="footer-cta__text">{{ contact_text }} </p>
+
+    <div class="footer-cta__grid">
+        {% include "patterns/atoms/avatar/avatar.html" with classes="footer-cta__avatar avatar--smallest" %}
+
+        <p class="footer-cta__cite">
+            <span class="footer-cta__name">{{ contact_name }}</span>
+            <span class="footer-cta__role">{{ contact_role }}</span>
```

Comment:
In the pattern lib, the spacing looks quite tight between these 2:

![image](https://github.com/torchbox/torchbox.com/assets/31627284/440d8046-0bd6-48d5-9eb5-36dbe1bdfdc1)

Design:

![image](https://github.com/torchbox/torchbox.com/assets/31627284/6c9b2dd8-da67-4523-a8f4-6e8534190a86)

Reply:
ah nice spot - fixed in 85c83e42db2fc924f4b1d4f42a85884af819aa0d